pwndbg/commands/killthreads.py

```diff
@@ -0,0 +1,52 @@
+import argparse
+from typing import List
+from typing import Optional
+
+import gdb
+
+import pwndbg.commands
+from pwndbg.color import message
+from pwndbg.commands import CommandCategory
+from pwndbg.gdblib.scheduler import lock_scheduler
+
+parser = argparse.ArgumentParser(description="Kill multiple threads at once.")
```

Comment:
Can we add a long description (and make sure newline works here, I think some other command like vmmap have description with newlines) and inform users on:
```
Kill all or given threads.

Switches to given threads and calls pthread_exit(0) on them.
This is performed with scheduler-locking to prevent other threads from operating at the same time.

Killing all other threads may be useful to use GDB checkpoints, e.g., to test given input & restart the execution to the point of interest (checkpoint).
```